### PR TITLE
Rewrote docs homepage

### PR DESCRIPTION
**Pull request by mattpocock**

### CODE REVIEW

File: docs/pages/docs/index.mdx [hunk 1]
```diff
@@ -9,22 +9,29 @@ import Features from "../../components/Features";
...
+## Features
+
+- **Never do the same work twice**: Turborepo will remember the output of tasks you (or your CI) ran and skip work that's already been done.
+- **Optimized scheduling**: Execute tasks using every core at maximum parallelism without wasting idle CPUs.
+- **Remote caching**: Share a remote cache across your organization for faster builds on CI/CD.
+- **Zero runtime overhead**: Turborepo doesn't interfere with your runtime code or touch your sourcemaps.
+
+## What turborepo is not
+
+Turborepo **doesn't handle package installation**. Tools like `npm`, `pnpm` or `yarn` already do that brilliantly. But they run tasks inefficiently, meaning slow CI builds.
```
Comment (tknickman) [inline docs/pages/docs/index.mdx:35]: They're slow outside of CI too 😄 

```suggestion
Turborepo **doesn't handle package installation**. Tools like `npm`, `pnpm` or `yarn` already do that brilliantly. But they run tasks slowly, and inefficiently.
```

**Review by tknickman — APPROVED** [review]

File: docs/pages/docs/index.mdx [hunk 2]
```diff
@@ -9,22 +9,29 @@ import Features from "../../components/Features";
...
-projects, often using multiple frameworks, in a single, unified
-code repository.
+Turborepo is a blazing fast build system for JavaScript/TypeScript monorepos - codebases containing multiple projects in a single, unified repository.
 
-Monorepos have many advantages, but they require appropriate tooling to scale. Turborepo provides this tooling and more. It leverages advanced build system ideas and techniques to speed up development, but with a fraction of the configuration and complexity. Turborepo abstracts all the annoying configuration, scripts, and tooling into `turbo.json`, so you can focus on _what_ you're shipping instead of worrying about _how_ it'll get shipped.
+## The problem
 
-Most importantly, unlike other build systems, Turborepo is designed to be incrementally adopted, so you can gradually and partially add it to most codebases in a few minutes.
+Monorepos have many advantages - but **they struggle to scale**. Each repository you add has its own test suite, its own build process and its own pipeline. A single monorepo might have **hundreds of tasks to perform**.
```
Comment (mehulkar) [inline docs/pages/docs/index.mdx:16]: - "Each repository you add" is confusing and may lead people down the path of thinking of submodules. "Each project you add" may be better
- "its own build process and its own pipeline" seems redundant. Also we may to reserve "pipeline" for the concept Turbo wants to introduce, rather than how a team already uses it.
- Mixing second and third persons here. "you add" (2nd person) vs "a single mono repo might have" (subject is 3rd person / thing)

File: docs/pages/docs/index.mdx [hunk 3]
```diff
@@ -9,22 +9,29 @@ import Features from "../../components/Features";
...
-- **Content-aware hashing**: Turborepo looks at the contents of your files, not timestamps, to figure out what needs to be built.
-- **Cloud caching**: Share a cloud build cache with your teammates and CI/CD for even faster builds.
-- **Parallel execution**: Execute builds using every core at maximum parallelism without wasting idle CPUs.
-- **Zero runtime overhead**: Turborepo doesn't interfere with your runtime code or touch your sourcemaps. It does what it does and then gets out of your way.
-- **Task pipelines**: Define the relationships between your tasks and then let Turborepo optimize what to build and when.
-- **Pruned subsets**: Speed up PaaS deploys by generating a subset of your monorepo with only what's needed to build a specific target.
-- **Convention-based config**: Reduce complexity through convention. Fan out configuration with just a few lines of JSON.
-- **Profile in your browser**: Generate build profiles and view them in a browser to understand which tasks are taking the longest.
+**Turborepo speeds up your monorepo**. Our remote cache stores the result of all your tasks, meaning that **your CI never needs to do the same work twice**.
```
Comment (mehulkar) [inline docs/pages/docs/index.mdx:20]:
```suggestion
**Turborepo speeds up the tasks in your monorepo**. Our remote cache stores the results of all your tasks, so that **your CI never needs to do the same work twice**.
```

Monorepos don't move themselves, so a tool can't speed them up 😛

File: docs/pages/docs/index.mdx [hunk 4]
```diff
@@ -9,22 +9,29 @@ import Features from "../../components/Features";
...
+Task scheduling can be a nightmare in a monorepo. Imagine `yarn build` needs to run before `yarn test`, across 40 workspaces. Turborepo **can schedule your tasks for maximum speed**, across all available cores.
+
+Turborepo can be **adopted incrementally**. It uses the `package.json` scripts you've already written, the dependencies you've already declared, and a single `turbo.json` file. You can **use it with any package manager**, like `npm`, `yarn` or `pnpm`. You can add it to any monorepo in just a few minutes.
+
+## Features
+
+- **Never do the same work twice**: Turborepo will remember the output of tasks you (or your CI) ran and skip work that's already been done.
+- **Optimized scheduling**: Execute tasks using every core at maximum parallelism without wasting idle CPUs.
+- **Remote caching**: Share a remote cache across your organization for faster builds on CI/CD.
+- **Zero runtime overhead**: Turborepo doesn't interfere with your runtime code or touch your sourcemaps.
```
Comment (mehulkar) [inline docs/pages/docs/index.mdx:31]: not part of this PR, but the "touch your sourcemaps" stuck out here. Is that something people are afraid of happening? Why would a monorepo tool ever affect sourcemaps?

File: docs/pages/docs/index.mdx (hunk 1 above)
Comment (mattpocock) [reply docs/pages/docs/index.mdx:35]: I think the main pain point for most users (and prompt to use turbo) is because they want to save CI time. Being more direct and less precise here IMO is better.

File: docs/pages/docs/index.mdx (hunk 3 above)
Comment (mattpocock) [reply docs/pages/docs/index.mdx:20]: I disagree - monorepos can be slow or fast!

File: docs/pages/docs/index.mdx (hunk 4 above)
Comment (mattpocock) [reply docs/pages/docs/index.mdx:31]: Yes, feels odd to me too.

File: docs/pages/docs/index.mdx (hunk 3 above)
Comment (mehulkar) [reply docs/pages/docs/index.mdx:20]: We may have to agree to disagree 😉 . Monorepos (by your own definition above) are:

> codebases containing multiple projects in a single, unified repository.

How can a codebase be fast or slow? Code can _run_ fast or slow and builds can _execute_ fast or slow; a codebase has no transformational aspect that has a speed!

Not a hill I want to die on though, obviously 😄  .

Comment (jaredpalmer) [reply docs/pages/docs/index.mdx:20]: Agree with @mehulkar here. Turbo also doesn't speed up the tasks, it runs them more efficiently and intelligently which results in faster builds. It explicitly doesn't touch the task itself.

File: docs/pages/docs/index.mdx (hunk 4 above)
Comment (jaredpalmer) [reply docs/pages/docs/index.mdx:31]: Our competitors' plugin rewrite framework outputs and change outputs directories and rewrite sourcemaps to get them to work within their constraints. This is vastly different that turbo.

File: docs/pages/docs/index.mdx (hunk 3 above)
Comment (mattpocock) [reply docs/pages/docs/index.mdx:20]: @jaredpalmer's take means that @mehulkar's suggestion (speeds up the tasks in your monorepo) is also not quite right.

I am still happy with "Turborepo speeds up your monorepo" as the best phrase (we've found so far) to represent the problem that turbo is solving.

"Turbo optimizes your monorepo" isn't specific enough
"Turbo optimizes how your monorepo runs tasks" doesn't address the _emotion_ behind the problem statement. "Monorepos are so slow" is exactly what folks are saying, why not reply with "Turborepo speeds them up".

Comment (mehulkar) [reply docs/pages/docs/index.mdx:20]: How about "Turborepo makes your monorepo smarter" ?

Comment (mattpocock) [reply docs/pages/docs/index.mdx]: @mehulkar I've gone with `Turborepo solves your monorepo's scaling problem`

File: docs/pages/docs/index.mdx [hunk 5]
```diff
@@ -9,22 +9,29 @@ import Features from "../../components/Features";
...
-- **Parallel execution**: Execute builds using every core at maximum parallelism without wasting idle CPUs.
-- **Zero runtime overhead**: Turborepo doesn't interfere with your runtime code or touch your sourcemaps. It does what it does and then gets out of your way.
-- **Task pipelines**: Define the relationships between your tasks and then let Turborepo optimize what to build and when.
-- **Pruned subsets**: Speed up PaaS deploys by generating a subset of your monorepo with only what's needed to build a specific target.
-- **Convention-based config**: Reduce complexity through convention. Fan out configuration with just a few lines of JSON.
-- **Profile in your browser**: Generate build profiles and view them in a browser to understand which tasks are taking the longest.
+**Turborepo speeds up your monorepo**. Our remote cache stores the result of all your tasks, meaning that **your CI never needs to do the same work twice**.
+
+Task scheduling can be a nightmare in a monorepo. Imagine `yarn build` needs to run before `yarn test`, across 40 workspaces. Turborepo **can schedule your tasks for maximum speed**, across all available cores.
```
[the remark this answers is not in the document]
Comment (mattpocock) [reply docs/pages/docs/index.mdx]: Addressed

File: docs/pages/docs/index.mdx (hunk 2 above)
Comment (mattpocock) [reply docs/pages/docs/index.mdx]: Addressed

**Review by mehulkar — APPROVED** [review]
Thanks for the updates! Looks like an improvement to me!
